### ISSUE TITLE
Feature Request: LLM Integration

### ISSUE THREAD

**Issue by ofseed** [mikesmithgh/kitty-scrollback.nvim#211]
I know actually this shouldn't be what this plugin should care about (so feel free to close it), but when I see how the user interacts with the paste window, I get this idea.

If I can write something I want to do in the paste window and this plugin generates the corresponding commands by LLMs, press enter, and the shell executes what I want to do. The final result is I can communicate with the shell in plain English!

**Comment by mikesmithgh** [comment]
Hey @ofseed, that is an interesting idea. This would be out of scope for this plugin since it is only opening the scrollback buffer. But, you could achieve this if you have an existing cli that gives the input/ouput and write that to the paste window, or possibly using another Neovim plugin that has LLM support and writing the results to the paste window.

Here is a fake example, `fake_llm` is just a script hardcoding the result.

I am opening kitty-scrollback.nvim, then pressing `a` to open the paste window. Then I enter the command `%!fake_llm list all the files in the current directory` to run an external command `fake_llm` and write the results to the paste window buffer.

https://github.com/mikesmithgh/kitty-scrollback.nvim/assets/10135646/9991bb51-e073-4924-b574-d94073b8705d
